sw/host/rom_ext_image_tools/signer/image/src/manifest.rs

```diff
@@ -0,0 +1,281 @@
+// Copyright lowRISC contributors.
```

Comment:
Can you include a comment indicating that this file is automatically generated?

Reply:
Done.